README.md

```diff
@@ -1,2 +1,7 @@
 # ipc-agent
 Implementation of an IPC agent
+
+## Dev
```

Comment:
Can we move this message somewhere else? Maybe into a `notes.md` in the root of the repo? I don't think it fits the main README.

Also, would you mind adding a bit more of context of where and how should the address be set for the first bullet point?